webui/react/src/pages/TrialDetails/TrialDetailsHyperparameters.tsx

```diff
@@ -57,18 +58,20 @@ const TrialDetailsHyperparameters: React.FC<Props> = ({ trial, pageRef }: Props)
 
   return (
     <div className={css.base}>
-      <InteractiveTable
-        columns={columns}
-        containerRef={pageRef}
-        dataSource={dataSource}
-        pagination={false}
-        rowClassName={defaultRowClassName({ clickable: false })}
-        rowKey="hyperparameter"
-        settings={settings as InteractiveTableSettings}
-        showSorterTooltip={false}
-        size="small"
-        updateSettings={updateSettings as UpdateSettings<InteractiveTableSettings>}
-      />
+      <Spinner conditionalRender spinning={!trial}>
```

Comment:
The `conditionalRender` keeps the "No Data" message from showing underneath the spinner-- that is good.

But it also makes it so where the layout changes more once the data is loaded, and it is not obvious that the thing that is being loaded is a table-- less than ideal.

It would be nice if we could still have the table skeleton stay in place underneath the loading spinner, but not say "No Data". It looks like we actually have a `SkeletonTable` that is not used... might be worth looking in to.